mediacapture-streams/GUM-deny.https.html

```diff
@@ -16,18 +16,16 @@ <h1 class="instructions">Description</h1>
   <script src=/resources/testharness.js></script>
   <script src=/resources/testharnessreport.js></script>
   <script>
-    var t = async_test("Tests that the error callback is triggered when permission is denied", {timeout:10000});
-    t.step(function() {
-      navigator.mediaDevices.getUserMedia({video: true})
-        .then(t.step_func(function (stream) {
-          assert_unreached("The success callback should not be triggered since access is to be denied");
-          t.done();
-        }), t.step_func(function (error) {
-          assert_equals(error.name, "NotAllowedError", "NotAllowedError returned as expected");
-          assert_equals(error.constraintName, undefined, "constraintName attribute not set as expected");
-          t.done();
-        }))
-    });
+    promise_test(t => {
+      return navigator.mediaDevices.getUserMedia({video: true})
+      .then(stream => {
+        assert_unreached("The success callback should not be triggered since access is to be denied");
+      })
+      .catch(error => {
+        assert_equals(error.name, "NotAllowedError", "NotAllowedError returned as expected");
+        assert_equals(error.constraintName, undefined, "constraintName attribute not set as expected");
+      });
```

Comment:
Use `promise_rejects(t, "NotAllowedError", () => navigator.mediaDevices.getUserMedia({video: true}))`

Reply:
@Ms2ger , thanks for your comment, fixed.